ros2doctor/ros2doctor/api/platform.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import platform
+import sys
+
+import rosdistro
+
+
+def print_platform_info():
+    """Print out platform information."""
+    # platform info
+    print('System Information')
+    print('system   : ', platform.system())
+    print('Platform Info   : ', platform.platform())
```

Comment:
nit, the spacing looks a little strange to me since the `:` don't line up. I recommend either removing the spaces before the `:` or making sure they're all aligned.

Something like 
```
System Information
system:  Linux
Platform Info:  Linux-4.15.0-55-generic-x86_64-with-Ubuntu-18.04-bionic
Mac OS version:  ('', ('', '', ''), '')
release:  4.15.0-55-generic
processor:  x86_64
Python
version:  3.6.8
compiler:  GCC 8.0.1 20180414 (experimental) [trunk revision 259383
build:  ('default', 'Jan 14 2019 11:02:34')
ROS Information
distribution name:  dashing
distribution type:  ros2
distribution status:  active
release platforms:  {'ubuntu': ['bionic']}
```

or
```
System Information
system               :  Linux
Platform Info        :  Linux-4.15.0-55-generic-x86_64-with-Ubuntu-18.04-bionic
Mac OS version       :  ('', ('', '', ''), '')
release              :  4.15.0-55-generic
processor            :  x86_64
Python
version              :  3.6.8
compiler             :  GCC 8.0.1 20180414 (experimental) [trunk revision 259383
build                :  ('default', 'Jan 14 2019 11:02:34')
ROS Information
distribution name    :  dashing
distribution type    :  ros2
distribution status  :  active
release platforms    :  {'ubuntu': ['bionic']}
```